evergreen.yml

```diff
@@ -0,0 +1,50 @@
+buildvariants:
+- display_name: Ubuntu 16.04
+  name: ubuntu1604
+  run_on:
+    - ubuntu1604-test
+  tasks:
+    - name: unit_tests
+    - name: check_formatting
+
+functions:
+  create virtualenv:
+  - command: shell.exec
+    params:
+      working_dir: src
+      script: |
+        set -o errexit
+        virtualenv --python=/opt/mongodbtoolchain/v3/bin/python3 venv
+        . venv/bin/activate
+
+        pip install -r requirements.txt
+        pip install -e .
+
+pre:
+- command: git.get_project
+  params:
+    directory: src
+- func: create virtualenv
+
+tasks:
+- name: unit_tests
+  commands:
+  - command: shell.exec
+    params:
+      working_dir: src
+      script: |
+        set -o errexit
+        . venv/bin/activate
+
+        pytest --flake8 --ignore=venv
```

Comment:
flake8 is checking dependency packages here (the evergreen logs [show](https://evergreen.mongodb.com/task_log_raw/selected_tests_ubuntu1604_unit_tests_patch_df063b57db3321f8255f98fc06e2e5f4fb998783_5d8bd9c02a60ed71a4405ab1_19_09_25_21_18_57/0?type=T#L88) a flake8 warning for the jinja2 package) and I'm not sure why

Reply:
It has to do with the way the flake8 analysis happens. Since jinja2 packages get imported into other files, they get analyzed. 

There is conversation about it [here](https://github.com/pallets/jinja/pull/867). It sounds like updating the version of flask we are using might help (if it has been released).

Reply:
I updated flask but it didn't fix it, so I'll leave it as-is for now